client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-instructions/steps/step-add-migration-key.tsx

```diff
@@ -0,0 +1,43 @@
+import clsx from 'clsx';
+import { useTranslate } from 'i18n-calypso';
+import { MigrationKeyInput } from '../migration-key-input';
+import type { FC } from 'react';
+
+interface Props {
+	migrationKey: string;
+	preparationError: Error | null;
+}
+
+export const StepAddMigrationKey: FC< Props > = ( { migrationKey, preparationError } ) => {
+	const translate = useTranslate();
+
+	if ( '' === migrationKey ) {
+		return (
+			<>
+				<p>{ translate( 'The key will be available here when your new site is ready.' ) }</p>
+				<div
+					className={ clsx( 'migration-key-skeleton', {
+						'migration-key-skeleton--animate': ! preparationError,
```

Comment:
It's just an extra tweak to not animate the field when an error happens. So the user doesn't have the sensation that something is still happening.

Reply:
The rest of this component didn't change. It was just extracted from `useSteps`.